### PR TITLE
Add Yeastar S-series VoIP PBX

### PR DESCRIPTION
**Pull request by walday**
Please give a short description what your pull request is for

DO NOT DELETE THE UNDERLYING TEXT

#### Please note

> Please read this information carefully. You can run `./lnms dev:check` to check your code before submitting.

- [ ] Have you followed our [code guidelines?](https://docs.librenms.org/Developing/Code-Guidelines/)
- [ ] If my Pull Request does some changes/fixes/enhancements in the WebUI, I have inserted a screenshot of it.
- [ ] If my Pull Request makes discovery/polling/yaml changes, I have added/updated [test data](https://docs.librenms.org/Developing/os/Test-Units/).

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

**Comment by murrant** [comment]
This is missing an icon and a test file with at minimum sysObjectID and sysDescr in it.

File: mibs/yeastar/YEASTAR-PBX-MIB-V4 [hunk 1]
```diff
@@ -0,0 +1,878 @@
+--
+-- YEASTAR-S-SERIES-V4.my
+-- MIB generated by MG-SOFT Visual MIB Builder Version 6.0  Build 88
+-- Wednesday, April 01, 2020 at 14:28:20
+--
+
+	YEASTAR-S-SERIES-V4 DEFINITIONS ::= BEGIN
```
Comment (PipoCanaja) [inline mibs/yeastar/YEASTAR-PBX-MIB-V4:7]: Please rename the MIB file as well, to name it "YEASTAR-S-SERIES-V4" as it is defined inside it.
Thanx

**Comment by Jellyfrog** [comment]
First of all, thank you for your contribution!

We're closing this due to inactivity, feel free to reopen if you find the time to continue on this!
